src/gamestates/enabledFunctions.lua

```diff
@@ -0,0 +1,24 @@
+local functionKeys =
```

Comment:
You could also call this `availableFunctions`.

Reply:
If you are talking about the function keys table then:
I prefer my name for that table since you need to enter them as strings.
Else if you are talking about the file:
Sure I can change the name if you like.

Reply:
I'm talking about the table `functionKeys`. "Key" isn't a very helpful name, since a key can be anything. `availableFunctions` makes more sense because it conotates a list of functions that can either be enabled or not enabled.

Reply:
These are keys to the love table and the gameState.
love[key]()

Reply:
These are okay with me:
```lua
enabledFunctionKeys
enabledFunctionKeyList
availableFunctionKeys
availableFunctionList
```

Reply:
Right, but anything could be a key to a table. Some other information about the contents of the table would be helpful. `{available,enabled}FunctionKeys` is fine. If you want to be clear that these are strings and not actual functions, you could call them `{available,enabled}FunctionNames`.